### ISSUE TITLE
VODCA v2: Multi-sensor, multi-frequency vegetation optical depth data for long-term canopy dynamics and biomass monitoring

### ISSUE THREAD

**Issue by puzhao8** [samapriya/awesome-gee-community-datasets#241]
### Contact Details

puzhao@kth.se

### Dataset description

download link: https://researchdata.tuwien.at/records/t74ty-tcx62

Citation:
Zotta, R.-M., Moesinger, L., van der Schalie, R., Vreugdenhil, M., Preimesberger, W., Frederikse, T., De Jeu, R., & Dorigo, W. (2024). VODCA v2: Multi-sensor, multi-frequency vegetation optical depth data for long-term canopy dynamics and biomass monitoring (1.0.0) [Data set]. TU Wien. https://doi.org/10.48436/t74ty-tcx62

VODCA v2
 
Vegetation optical depth (VOD) is a model-based indicator of the water content stored in the vegetation canopy and is derived from microwave Earth observations. Moesinger et al. (2020) introduced the Global Microwave Vegetation Optical Depth Climate Archive (VODCA v1; [10.5281/zenodo.2575598](https://zenodo.org/doi/10.5281/zenodo.2575598)), a dataset that harmonizes Vegetation Optical Depth (VOD) retrievals from multiple sensors across the C-, X-, and Ku frequency bands. This archive comprises three long-term, multi-sensor, single-frequency VOD products, covering over 30 years of observations. 

VODCA v2 incorporates several methodological improvements compared to the first version and adds two new VOD datasets to the VODCA product suite. The VOD observations used in VODCA v2 are derived through the Land Parameter Retrieval Model (LPRM; Owe et al. (2008); Van der Schalie et al. (2017))

VODCA CXKu is a multi-sensor, multi-frequency product of unprecedented coverage (34 years; 1987 - 2021), computed using observations in the C-, X- and Ku-band frequencies from the following sensors: the Special Sensor Microwave Imager (SSM/I) F08, F11, F13, F17, the Tropical Rainfall Measuring Mission (TRMM) Microwave Imager (TMI), The Advanced Microwave Scanning Radiometer – Earth Observing System (AMSR-E), Windsat, the Advanced Microwave Radiometer 2 (AMSR2) and the Global Precipitation Measurement (GPM) Microwave Imager (GMI). We harmonized observations from the different sensors and frequencies by scaling them to AMSR-E X-band and then using a weighted averaging technique to fuse overlapping observations. VODCA CXKu is available at a daily temporal resolution and a spatial resolution of 0.25°. The data is masked for spurious observations (e.g., frozen ground, snow, radio-frequency interference). 

VODCA L is a single-frequency, L-band product created by using VOD observations from the Soil Moisture and Ocean Salinity (SMOS) Microwave Imaging Radiometer using Aperture Synthesis (MIRAS) and Soil Moisture Active Passive (SMAP) radiometer. We harmonized observations from the two sensors by scaling SMOS to SMAP and using a weighted averaging technique for the overlapping observations. VODCA L (2010 - 2021) has 10 daily observations and a spatial resolution of 0.25°. The data is masked for spurious observations (e.g., frozen ground, snow, radio-frequency interference). 

Technical details
Files: 

"VODCA_CXKu.zip" (unzipped size: ~95 GB)
VODCA_CXKu daily images stored in netCDF (.nc) format, sorted into yearly folders
"VODCA_L.zip" (unzipped size: ~3.5 GB)
VODCA_L 10 daily images stored in netCDF (.nc) format, sorted into yearly folders
Variables: 

for VODCA CXKU:
VODCA_CXKu - Unitless, Vegetation Optical Depth (multi-sensor, multi-frequency VOD from the C-, X- and Ku-band frequencies)
"time"/"lon"/"lat": Dimensions of the data
 

for VODCA L:
VODCA_L:  Unitless, Vegetation Optical Depth (multi-sensor VOD in the L-band frequency)
"time"/"lon"/"lat": Dimensions of the data
References:

Moesinger, L., Dorigo, W., de Jeu, R., van der Schalie, R., Scanlon, T., Teubner, I., and Forkel, M.: The global long-term microwave Vegetation Optical Depth Climate Archive (VODCA), Earth Syst. Sci. Data, 12, 177–196, https://doi.org/10.5194/essd-12-177-2020, 2020. 

Van der Schalie, Robin, et al. "The merging of radiative transfer based surface soil moisture data from SMOS and AMSR-E." Remote Sensing of Environment 189 (2017): 180-193.

Owe, Manfred, Richard de Jeu, and Thomas Holmes. "Multisensor historical climatology of satellite‐derived global land surface moisture." Journal of Geophysical Research: Earth Surface 113.F1 (2008).

### Earth Engine Snippet if dataset already in GEE

for example
```js
var asset_name = ee.ImageCollection("path to your collection")
```
Sample Code: Add a sample code maybe just adding your datasets in the code editor

### Enter license information

Enter License information ex: APACHE 2.0, CC-BY-4.0

### Keywords

VOD, VODCA, Radar, Multi-Sensor, optical depth

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by harrybmwells** [comment]
Yes please! This dataset would be valuable addition. Many thanks in advance!

**Comment by samapriya** [comment]
This has been completed and will be made available next week . Thanks for being patient and for contributing
